flutter_news_example/lib/app/router/app_router.dart

```diff
@@ -33,8 +40,15 @@ class AppRouter {
     GoRoute(
       path: LoginPage.path,
       name: LoginPage.path,
-      pageBuilder: (context, state) => const MaterialPage<void>(
-        child: LoginPage(),
+      builder: (context, state) => SignInScreen(
+        auth: Platform.environment.containsKey('FLUTTER_TEST')
+            ? MockAuth()
+            : null,
+        providers: [
+          EmailAuthProvider(),
+          // TODO(any): add client id to enable desktop login
+          GoogleProvider(clientId: ''),
```

Comment:
What is the google provider `clientId` for?

Reply:
This is to enable client id in desktop, in the docs said that for android and iOS is not necessary

Reply:
the field is required, for that reason I left it with clientId, if we remove the class we don't have the google sign in button